### PR TITLE
Bumping k8s version to 1.13.0-beta.1; Pointing to kubernetes-sigs/sig-storage-lib-external-provisioner instead of external-storage; Update CSINodeInfo schema

### PR DESCRIPTION
**Pull request by verult**
Note for reviewers: version bump and provisioner lib change are combined into the first commit. See Gopkg.toml and all changes in `cmd` and `pkg`.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -38,32 +38,33 @@
 
 [[constraint]]
   name = "github.com/kubernetes-csi/external-snapshotter"
-  version = "1.0.0-rc2"
+  #version = "1.0.0-rc2"
+  branch = "master"
```
Comment (msau42) [inline Gopkg.toml]: why does this need to be master?

Comment (verult) [reply Gopkg.toml]: Ah ok didn't realize rc3 has been cut, will update

**Pull request merged**